src/panel/direction/DirectionInput.jsx

```diff
@@ -23,11 +24,21 @@ class DirectionInput extends React.Component {
   }
 
   componentDidMount() {
+    if (this.props.isLoading) {
+      this.props.inputRef.current.blur();
```

Comment:
for instance, `isLoading` is true when the app is restoring origin and destination points from the url after a reload[...]